ui/send_page.go

```diff
@@ -402,7 +402,10 @@ func (pg *sendPage) fromSection(gtx layout.Context, common pageCommon) layout.Di
 							return layout.E.Layout(gtx, func(gtx C) D {
 								return layout.Flex{}.Layout(gtx,
 									layout.Rigid(func(gtx C) D {
-										return pg.theme.Body1((*pg.selectedAccount).TotalBalance).Layout(gtx)
+										spendableBalance := dcrutil.Amount((*pg.selectedAccount).SpendableBalance).String()
+										txt := pg.theme.Body1(spendableBalance)
+										txt.Color = pg.theme.Color.DeepBlue
```

Comment:
The default text colour for body1 text is already DeepBlue. There's no need to set the value again in the the code.